### ISSUE TITLE
[Feature request] Sync settings between devices

### ISSUE THREAD

**Issue by erikgeurts** [leggett/simplify#598]
I was very happy to notice that I was able to set up a subscription recently, and I did so immediately.

Given that I use multiple devices, based on what I'm working on and where I'm working at, it would be great if it were possible to have Simplify setting sync between devices, so that when I make a change somewhere, it's picked up elsewhere without having to think about it.

**Comment by leggett** [comment]
Absolutely. I have this high on my to do list. And thanks for signing up!!

**Comment by jahirfiquitiva** [comment]
Looking forward to this feature 🤞

**Comment by ben-foxmoore** [comment]
Is there any less-than-user-friendly solution to this? It's quite painful at the moment when switching between devices!

**Comment by leggett** [comment]
I'm working on this now. I have documentation on a manual way to export/import settings but that doesn't keep them synced and it isn't very user friendly.

**Comment by ben-foxmoore** [comment]
Any chance of being an early tester of those manual instructions? My current use case is switching browsers on the same device, so it's hopefully a one time thing anyway!

**Comment by leggett** [comment]
If you are looking to do a one-time import/export and are comfortable with the developer console, email me at hi@simpl.fyi and I'd be happy to share my instructions.
